.github/workflows/update-test.yaml

```diff
@@ -9,16 +9,19 @@ on:
   pull_request:
 jobs:
   update_test:
-    name: Update test PG${{ matrix.pg }}
+    name: Update test PG${{ matrix.pg }} ${{ matrix.kind }}
     runs-on: 'ubuntu-18.04'
     strategy:
       matrix:
         pg: ["11.10","12.5"]
+        opt: ["", "-r"]
         include:
           - pg: 11.10
             pg_major: 11
           - pg: 12.5
             pg_major: 12
+          - opt: "-r"
```

Comment:
Do we need to run this for both PG versions? Maybe it is enough to just run with PG12?

Reply:
There are optimizations to do, but I would prefer to defer that to a separate PR.